src/TesterInternal/StorageTests/PersistenceGrainTests_AzureStore.cs

```diff
@@ -24,11 +23,9 @@
 namespace UnitTests.StorageTests
 {
     /// <summary>
-    /// PersistenceGrainTests using AzureStore - Requires access to external Azure storage
+    /// PersistenceGrainTests using AzureStore - Requires access to external Azure table storage
```

Comment:
I thought you turned this as a base class for both table and blob storage tests... why is the comment being made specific to say "table"?